insert.js

```diff
@@ -0,0 +1,33 @@
+var mongoose = require('mongoose'); //require mongoose native drivers
+var url = 'mongodb://localhost/vaibhav'; //connection url of the database
+var conn = mongoose.connect(url); //use the connect method to conenct to the server
+
+var post_schema = mongoose.Schema({}, {
```

Comment:
the structure you are follow is incorrect. 
use the correct structure of schema.
var thingSchema = new Schema(
{
name: String,
dob: Number,
gender: String
}, {
strict:true
 
});